scripts/us_census/pep/annual_population/states_to_shortform.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This Script converts States Full Form to Short Form"""
+_STATES = {
```

Comment:
how is this different from https://github.com/datacommonsorg/data/blob/master/util/name_to_alpha2.py#L16 ?